i18n/en.jsonp.js

```diff
@@ -178,6 +178,11 @@ document.localeJson = {
       "dialog-readzim-error-message": "Error reading ZIM file",
       "dialog-reset-warning-title": "WARNING!",
       "dialog-reset-warning-message": "This will reset the app to a freshly installed state, deleting all app caches and settings!",
+      "dialog-metadata-warning": "Warning: above data can be spoofed!",
+      "dialog-metadata-name": "Name: ",
+      "dialog-metadata-creator": "Creator: ",
+      "dialog-metadata-publisher": "Publisher: ",
+      "dialog-metadata-scraper": "Scraper: ",
```

Comment:
Please move this block above so it is immediately below the last `dialog-messagechannel...` translation (so it's in roughly alphabetical order).